tests/testthat/test-fetch.R

```diff
@@ -0,0 +1,63 @@
+test_that('Test fetchMetadata', {
+  skip_on_os('windows', arch = NULL)
+  originalColumns = c('study_accession', 'sample_accession', 'secondary_sample_accession',
+                      'sample_alias', 'sample_title', 'experiment_accession',
+                      'run_accession', 'fastq_md5', 'fastq_ftp', 'fastq_aspera')
+  metadataControl = metadata[, ..originalColumns]
```

Comment:
For consistency with other packages, please use suffix `Exp` instead of `Control`.

Reply:
Done, even on salmon output directory in data directory.